src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -243,25 +243,46 @@ private String get_user_myworkspace_site_json(String userEid, String sessionId)
 		String errorMessage = "";
 		String requestUrl = "";
 
-		// get all sites that user is of Owner role
+		// First, get all sites that user is of Owner role
+		JSONArray sitesJSONArray = getSitesJSONArray(currentUserId, sessionId, allowedSiteTypes, Utils.ROLE_OWNER);
+		// Then append all sites that user is of Instructor role
+		sitesJSONArray.put(getSitesJSONArray(currentUserId, sessionId, allowedSiteTypes, Utils.ROLE_INSTRUCTOR));
+
+		JSONObject sitesJSONObject = new JSONObject();
+		sitesJSONObject.put(JSON_ATTR_SITE_COLLECTION, sitesJSONArray);
+
+		rv.put("projectsString", sitesJSONObject.toString());
+		rv.put("errorMessage", errorMessage);
+		rv.put("requestUrl", requestUrl);
+		return rv;
+	}
+
+	/**
+	 * Util class used for get site JSONArray based on role specified
+	 * @param currentUserId
+	 * @param sessionId
+	 * @param allowedSiteTypes
+	 * @param role
+	 * @return
+	 */
+	private JSONArray getSitesJSONArray(String currentUserId, String sessionId, List<String> allowedSiteTypes, String role) {
+		String requestUrl;
+		String errorMessage;
+		JSONArray sitesJSONArray = new JSONArray();
 		RestTemplate restTemplate = new RestTemplate();
-		requestUrl = Utils.directCallUrl(env, "membership.json?role=" + Utils.ROLE_OWNER + "&", sessionId);
+		requestUrl = Utils.directCallUrl(env, "membership.json?role=" + role + "&", sessionId);
 		log.info(this + " get_user_sites " + requestUrl);
 		try {
 			String membershipString = restTemplate.getForObject(requestUrl,
 					String.class);
 
 			// update the projectString by filtering based on site Owner role
```

Comment:
is this comment still valid w.r.t to owner role? as I see code seems to allow other roles as well like instructor